core-aam/aria-hidden_value_changes-manual.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html>
+  <head>
+    <title>aria-describedby value changes</title>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" href="/wai-aria/scripts/manual.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/wai-aria/scripts/ATTAcomm.js"></script>
+    <script>
+    setup({explicit_timeout: true, explicit_done: true });
+
+    var theTest = new ATTAcomm(
+    {
+   "steps" : [
+      {
+         "attribute" : "aria-hidden",
+         "element" : "test",
+         "title" : "step 1",
+         "type" : "attribute",
+         "value" : "true"
+      },
+      {
+         "element" : "test",
+         "test" : {
+            "ATK" : [
+               [
+                  "event",
+                  "type",
+                  "is",
+                  "object:property-change:aria-hidden"
+               ]
+            ],
+            "AXAPI" : [
+               [
+                  "event",
+                  "type",
+                  "is",
+                  "AXUIElementDestroyed"
```

Comment:
I'm not sure this event happens "on" the object that is being destroyed, of it it is associated with the document some other way.

Reply:
Have you been able to test it and see what happens?

Reply:
I just confirmed with a VO engineer that it’s, “on the object.”